### ISSUE TITLE
Question: What type of aria-live addition should be added for translation progress updates?

### ISSUE THREAD

**Issue by mejackreed** [projectblacklight/spotlight#1962]
#1961 adds translation progress dynamically. There was questions about whether or not this should have aria-live enhancements added.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions

cc @ggeisler

**Comment by ggeisler** [comment]
The progress indicators added in #1961 are **not** dynamic, if I understand @camillevilla's comment in the PR correctly. Is that correct, versus @mejackreed's statement above that they are dynamic?

Since the form input entries are not saved until the Save button is selected, I don't think we need to save the progress dynamically (that is, as each input is filled in, before saving the form). If the progress indicator was a more central part of the form then I think dynamic updating would be more important. With the indicators in the tabs, the status is more for the user to understand which tab might need attention, not so much to show step-by-step progress as the user is filling out a specific form.

**Comment by jkeck** [comment]
The concern that brought this up is that the progress is initially inserted via javascript (which was ultimately much simpler than trying to do it in the view layer).

So the page render cycle renders the tabs first, the tab content second, and the updates the progress later on. (**Not dynamically** as the user fills out inputs)

We weren't entirely sure if we there was any accessibility concern w/ the approach we were taking, so wanted to add an issue so that we didn't block #1961.

**Comment by ggeisler** [comment]
Oh okay. I think I was confused by "adds translation progress dynamically" which sounded more like step-by-step progress than "adds translation progress indicators dynamically" would have.

So I understand the concern now, though I'm not sure I have an answer to whether this approach presents an accessibility concern. I think it might depend on how quickly the progress indicator is added by the javascript and whether a screenreader would know about the indicator without resorting to the live region approach as you all were considering.

I can test things out using a screenreader with the #1961 code and update this ticket on what happens.
